test/PayID/pay-id-utils-test.ts

```diff
@@ -1,66 +1,66 @@
 import { assert } from 'chai'
 import PayIDUtils from '../../src/PayID/pay-id-utils'
 import 'mocha'
-import PaymentPointer from '../../src/PayID/payment-pointer'
 
 describe('PayIDUtils', function(): void {
-  it('parse payment pointer - host and path', function(): void {
-    // GIVEN a payment pointer with a host and a path.
-    const rawPaymentPointer = '$example.com/foo'
+  it('parse Pay ID - valid', function(): void {
+    // GIVEN a Pay ID with a host and a path.
+    const host = 'xpring.money'
+    const path = 'georgewashington'
+    const rawPayID = `${path}$${host}`
 
-    // WHEN it is parsed to a PaymentPointer object
-    const paymentPointer = PayIDUtils.parsePaymentPointer(rawPaymentPointer)
+    // WHEN it is parsed to components.
+    const payIDComponents = PayIDUtils.parsePayID(rawPayID)
 
     // THEN the host and path are set correctly.
-    assert.equal(paymentPointer?.host, 'example.com')
-    assert.equal(paymentPointer?.path, '/foo')
+    assert.equal(payIDComponents?.host, host)
+    assert.equal(payIDComponents?.path, `/${path}`)
   })
 
-  it('parse payment pointer - well known path', function(): void {
-    // GIVEN a payment pointer with a well known path.
-    const rawPaymentPointer = '$example.com'
+  it('parse Pay ID - multiple dollar signs', function(): void {
+    // GIVEN a Pay ID with too many '$'.
```

Comment:
I'm not sure this test is accurate. You _can_ have `$`s in the `path` of a URL. Maybe we should say the PayID spec disallows that, but I'm not sure it does right now.

Thoughts @austin-king ?

Reply:
I think `$` should be a reserved char

Reply:
Think it's best to disallow multiple $'s for PayID. I imagine this would be very confusing to any user.